doc/srcery.txt

```diff
@@ -40,17 +40,25 @@ sessions using an editor or terminal emulator.
 
 ==============================================================================
 INSTALLATION						   *srcery-installation*
-
 Put srcery.vim in `~/.vim/colors/` (on unix-like systems) or
-`%userprofile%\vimfiles\colors\` (on Windows).
+`%userprofile%\vimfiles\colors\` (on Windows). You can also install with your
+favourite plugin manager.
+
+Don't forget to set your 'runtimepath' correctly, otherwise srcery will not
+work as it relies on 'autoload' functionality.
 
 							   *srcery-install-vim8*
 
-Vim 8 has native support for loading plugins. All you need to do to is to
-clone this repository into `~/.vim/plug/default/opt`.
+Vim 8 has native support for loading plugins by using |packages|. All you need
+to do to is to clone this repository into `~/.vim/pack/themes/opt`.
```

Comment:
```suggestion
Vim 8 has native support for loading plugins by using |packages|. All you need
to do, is to clone this repository into `~/.vim/pack/themes/opt`.
```

My comma suggestion might be wrong, though. 🤣

Reply:
Solved in 398a4e264119b8c8c09e6926733b100b054f15a0